### PR TITLE
[NR-65072] The chart readme was not covering the correct default 

### PR DESCRIPTION
**Pull request by paologallinaharbur**
Just fixing the chart docs to cover default usecase

### CODE REVIEW

**Comment by sigilioso** [comment]
> Just fixing the chart docs to cover default usecase

Removing the docs which are published in the website is also intended, right?

**Review by sigilioso — APPROVED** [review]
👍

File: README.md [hunk 1]
```diff
@@ -35,17 +35,20 @@ helm upgrade --install newrelic newrelic-prometheus/newrelic-prometheus-agent -f
 For further information of the configuration needed for the chart just read the [chart's README](/charts/newrelic-prometheus-agent/README.md).
 
 ## Getting Started
-The simplest way to start working with this project is to install the [Chart](/charts/newrelic-prometheus-agent/README.md) in your Kubernetes cluster which by default will scrape the Prometheus targets having the `'prometheus.io/scrape: true'` annotation.
+The simplest way to start working with this project is to install the [Chart](/charts/newrelic-prometheus-agent/README.md) in your Kubernetes cluster.
+
+All pod and endpoints with the `newrelic.io/scrape: true` annotation will be scraped by default.
+
+Moreover, the solution will scrape as well all pods and endpoints with the `prometheus.io/scrape: true` annotations and
+having one of the labels matching the integrations_filter configuration.
```
Comment (marcsanmi) [inline README.md:43]:
```suggestion
have one of the labels matching the integrations_filter configuration.
```

File: README.md [hunk 2]
```diff
@@ -35,17 +35,20 @@ helm upgrade --install newrelic newrelic-prometheus/newrelic-prometheus-agent -f
 For further information of the configuration needed for the chart just read the [chart's README](/charts/newrelic-prometheus-agent/README.md).
 
 ## Getting Started
-The simplest way to start working with this project is to install the [Chart](/charts/newrelic-prometheus-agent/README.md) in your Kubernetes cluster which by default will scrape the Prometheus targets having the `'prometheus.io/scrape: true'` annotation.
+The simplest way to start working with this project is to install the [Chart](/charts/newrelic-prometheus-agent/README.md) in your Kubernetes cluster.
+
+All pod and endpoints with the `newrelic.io/scrape: true` annotation will be scraped by default.
```
Comment (marcsanmi) [inline README.md:40]:
```suggestion
All pods and endpoints with the `newrelic.io/scrape: true` annotation will be scraped by default.
```

File: README.md [hunk 3]
```diff
@@ -35,17 +35,20 @@ helm upgrade --install newrelic newrelic-prometheus/newrelic-prometheus-agent -f
...
 ### Usage
-Content: 
-- [Filter targets in Kubernetes](/docs/KuberntesTargetFilter.md)
-- [Filter metrics and labels](/docs/MetricsFilters.md)
-- [Target authorization](/docs/TargetAuthorization.md)
-- [Scrape Interval](/docs/ScrapeInterval.md)
-- [Metrics Labels/Metadata details](/docs/MetricLabels.md) 
-- [POMI migration guide](/docs/MigrationGuide.md)
-- [Troubleshooting guide](/docs/Troubleshooting.md)
+For further information checkout [the official documentation](https://docs.newrelic.com/docs/infrastructure/prometheus-integrations/install-configure-prometheus-agent/install-prometheus-agent/)
```
Comment (marcsanmi) [inline README.md:50]:
```suggestion
For further information check out [the official documentation](https://docs.newrelic.com/docs/infrastructure/prometheus-integrations/install-configure-prometheus-agent/install-prometheus-agent/) 
```

**Review by marcsanmi — APPROVED** [review]
👍 Just left some spelling errors.

File: README.md (hunk 3 above)
Comment (paologallinaharbur) [reply README.md:50]: oh, you are right https://dictionary.cambridge.org/dictionary/english/checkout
I think I have been always using it as a verb 😖

File: README.md (hunk 1 above)
Comment (paologallinaharbur) [reply README.md:43]: 🤔 I believe in this case it is correct. 

it would be have if it was " and `that` have one of the..."

**Pull request merged**